spec/swagger.yaml

```diff
@@ -4727,7 +4716,7 @@ paths:
                     details: null
   /jobs-api/v3/projects/{projectId}/jobs/{translationJobUid}/file/remove:
     post:
-      summary: "Remove a file from a job"
```

Comment:
Just wandering, why did you removed all articles from the summaries?

Reply:
Decided to keep descriptions as concise as possible.